docs/comit-protocol/comit-projects.md

```diff
@@ -0,0 +1,32 @@
+---
+id: comit-projects
+title: COMIT projects
+sidebar_label: COMIT projects
+---
+
+⚠️ Disclaimer: This part of the documentation is still under construction!
+
+This section of the documentation provides an overview of the different projects of the COMIT protocol.
+Sub-sections provide an introduction into project-specific details.
+
+The COMIT protocol is completely open source and consists of several repositories on GitHub.
+Here is a list of the most important repositories on GitHub:
+
+* [comit-rs](https://github.com/comit-network/comit-rs): The comit-rs project is the reference implementation of the protocol written in Rust. The [comit network daemon (cnd)](comit-protocol-stack.md#comit-network-daemon-cnd) is the binary that is released by the comit-rs project.
+* [comit-js-sdk](https://github.com/comit-network/comit-js-sdk): The COMIT Javascript SDK (comit-js-sdk) wraps the communication with cnd and provides an entry point for integrating atomic swaps with COMIT into Javascript applications. The comit-js-sdk is written Typescript.
+* [create-comit-app](https://github.com/comit-network/create-comit-app): The create-comit-app project includes Typescript demos and examples that show how to use the comit-js-sdk. Additionally it includes the `start-env` which spins up a development environment using docker containers.
```

Comment:
I'd link to the tutorial that explains how to use it.

Reply:
Does that mean I should do something? I am not sure I understand the comment :)

Reply:
Please add a link to the create-comit-app tutorial :)

Reply:
got it :)